Marlin/src/inc/SanityCheck.h

```diff
@@ -515,6 +515,10 @@
   #error "DIGIPOT_I2C is now DIGIPOT_MCP4451 (or DIGIPOT_MCP4018). Please update Configuration_adv.h."
 #elif defined(TOUCH_BUTTONS)
   #error "TOUCH_BUTTONS is now TOUCH_SCREEN. Please update your Configuration.h."
+#elif defined(LCD_FULL_PIXEL_HEIGHT)
+  #error "LCD_FULL_PIXEL_HEIGHT is now TFT_HEIGHT. Please update your Configuration.h."
+#elif defined(LCD_FULL_PIXEL_WIDTH)
+  #error "LCD_FULL_PIXEL_WIDTH is now TFT_WIDTH. Please update your Configuration.h."
```

Comment:
These have never been configurable options, but come from the type of LCD selected, so there's no need for a sanity-check.

Reply:
Just added it, because some users fork may have added it to use emulated dogm.
But, the default done by scaled_tft.h may solve it.
So, yes, we can remove those checks.

Reply:
From what you say, the checks should stay, but they should tell the user to remove them if they exist.